vars/gradleCheckFlakyTestDetector.groovy

```diff
@@ -28,7 +29,8 @@ void call(Map args = [:]) {
             def awsAccessKey = env.AWS_ACCESS_KEY_ID
             def awsSecretKey = env.AWS_SECRET_ACCESS_KEY
             def awsSessionToken = env.AWS_SESSION_TOKEN
```

Comment:
Where are we setting these values? Should be coming from secrets right?

Reply:
Yes Sayali the values are coming from secret, `withAWS(role: 'OpenSearchJenkinsAccessRole', roleAccount: "${METRICS_HOST_ACCOUNT}", duration: 900, roleSessionName: 'jenkins-session')` when used `withAWS` they are by default added to the `env` and the library uses those to create the sigv4 request.

Reply:
I see, is there are possibility of exposing those secrets? I know, jenkins take care of everything that is within `withCredentials` block, but these env variables are coming as a subset of that. 
Just want to confirm we are not exposing the credentials. (even though temporary) while doing curl or other activities.

Reply:
It wont be exposed Sayali, also we use `set +x` which disables debugging mode in the shell [OpenSearchMetricsQuery.groovy](https://github.com/opensearch-project/opensearch-build-libraries/blob/main/src/gradlecheck/OpenSearchMetricsQuery.groovy)., here is an example build https://build.ci.opensearch.org/job/gradle-check-flaky-test-detector/55/console.